content/articles/2018-11-tibble-2.0.0-pre-announce.Rmd

```diff
@@ -129,41 +129,56 @@ Packages that are in the business of making tibbles may even want to expose the
 
 ## Deprecation of `validate` in `as_tibble()`
 
-In tibble 1.4.2, `as_tibble()` has a `validate` argument, but its default behaviour value was inconsistent across different methods and there was no equivalent argument for `tibble()`. The `validate` argument is now soft-deprecated and its use will trigger a message (we intend to make this "once per session"). The `validate` argument will eventually be removed.
+In tibble v1.4.2, `as_tibble()` has a `validate` argument, but its default behaviour value was inconsistent across different methods and there was no equivalent argument for `tibble()`. The `validate` argument is now soft-deprecated and its use will trigger a message, once per session. The `validate` argument will eventually be removed, but for now it can be used jointly with the new `.name_repair` argument (without even triggering a message). This is possible, because fortunately the `.name_repair` argument to `as_tibble()` is ignored in tibble v1.4.2.
```

Comment:
Is this the point: revdep maintainer can release a version that works for v1.4.2 and v2.0.0, *even before tibble v2.0.0 releases*, by using both `validate` and `.name_repair`? You are describing a transition plan.

If I've got that right, does my wording suggest any changes you are OK with?

Reply:
Yes. I like the term "transition plan". I don't understand "suggest any changes" -- happy to use your preferred wording.